app/views/layouts/_header.html.erb

```diff
@@ -67,7 +66,7 @@
     <div class="col-md-3 header-right">
       <div class="search-bar">
         <input type="text" value="Search" onfocus="this.value = '';" onblur="if (this.value == '') {this.value = 'Search';}">
```

Comment:
`onblur="if (this.value == '') {this.value = 'Search';}"`
tạo file js để xử lý em nhé